emulation_configure.bash

```diff
@@ -193,8 +193,6 @@ function verify_host_environment() {
     [ $LOOPS -gt 0 ] && die \
     	'losetup -al shows active loopback mounts, please clear them'
 
-    set -- `qemu-system-x86_64 -version`
```

Comment:
This check needs to stay, but be augmented to reflect all versions for which this script produces viable output.   Put back the check for 2.6.0 and add a check for your 2.8.0.
Yes, skip 2.7.0 as we don't know that it works.